app/views/admin/configs/show.html.erb

```diff
@@ -324,7 +324,7 @@
                             <div
                               id="<%= provider.provider_name %>-enabled-indicator"
                               class="enabled-indicator <%= authentication_provider_enabled?(provider) ? "visible" : "" %>">
-                              <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon admin-config-checkmark", title: "Checkmark") %>
+                              <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon admin-config-checkmark", title: "#{provider.official_name} enabled") %>
```

Comment:
The change is for accessibility, but it also simplifies the test by getting by title via Cypress testing library APIs.